src/components/Projects.tsx

```diff
@@ -1,7 +1,214 @@
 import React from "react";
+import styled from "styled-components";
+import { Link } from "react-router-dom";
+import {
+  space,
+  SpaceProps,
+  layout,
+  LayoutProps,
+  background,
+  BackgroundProps,
+  FlexboxProps,
+  flexbox,
+  PositionProps,
+  position,
+  ColorProps,
+  color,
+  TypographyProps,
+  typography,
+  border,
+  BorderProps,
+} from "styled-system";
+import stairs from "./assets/project-page/stairs.png";
+import shell from "./assets/project-page/shell.png";
+import eye from "./assets/project-page/eye.png";
+import statue from "./assets/project-page/statue.png";
+import dragon from "./assets/project-page/dragon.png";
+import knife from "./assets/project-page/knife.png";
+import mask from "./assets/project-page/mask.png";
+import spider from "./assets/project-page/spider.png";
+import magnify from "./assets/project-page/magnify.png";
+
+type BgProps = BackgroundProps &
+  ColorProps &
+  LayoutProps &
+  FlexboxProps &
+  TypographyProps &
+  LayoutProps &
+  SpaceProps;
+
+const Background = styled.div<BgProps>`
+  ${background};
+  ${color};
+  ${layout};
+  ${flexbox};
+  ${space};
+`;
+
+type LogoProps = SpaceProps &
+  LayoutProps &
+  FlexboxProps &
+  PositionProps &
+  ColorProps &
+  SpaceProps &
+  TypographyProps &
+  BorderProps &
+  ColorProps;
+
+const HomeLink = styled.a<LogoProps>`
+  ${space};
+`;
+
+const HomeLogo = styled.p<LogoProps>`
+  ${flexbox};
+  ${typography}
+  ${color}
+`;
+
+const BuyButton = styled.button<LogoProps>`
+  ${space};
+  ${layout};
+  ${color};
+  ${typography};
+  ${flexbox};
+  ${border}
+  ${position};
+`;
+
+const Icon = styled.a<LogoProps>`
+  ${layout};
+  ${position};
+  ${flexbox};
+  ${space};
+  ${position};
+  a:hover {
+    width: 2em;
+  }
+`;
 
 const Projects = () => {
-  return <div>Projects</div>;
+  return (
+    <Background
+      backgroundSize="cover"
+      backgroundImage="url('./assets/home-page/background.jpg')"
+      backgroundPosition="center"
+      backgroundRepeat="no-repeat"
+      bg="black"
+      height="100vh"
+      display="flex"
+      flexDirection="column"
+      alignItems="center"
+      mx="auto"
+    >
+      <HomeLink p={30}>
+        <Link to="/" style={{ textDecoration: "none" }}>
+          <HomeLogo
+            fontFamily="SangBleu OG Serif Light Regular"
+            color="athensGray"
+            fontSize={[3, 3, 3, 4, 5]}
+          >
+            OXYMORE
+          </HomeLogo>
+        </Link>
+      </HomeLink>
+      <Icon
+        width={[50, 60, 60, 100, 120]}
+        position="absolute"
+        top={[80, 80, 80, 180, 180]}
+        left={[160, 160, 180, 120, 160]}
+      >
+        <img src={stairs}></img>
+      </Icon>
+
+      <Icon
+        width={[50, 60, 60, 100, 120]}
+        position="absolute"
```

Comment:
Since the `width` and `position` are the same for all of these, could they go in the styled-components definition for `Icon` above? That keeps it DRY (you heard that acronym?)

Also, could `Icon` be `ProjectIcon` for a little more clarity?

Reply:
> You can add breakpoints in `src/components/theme.ts`.
> 
> However, for this you may or may not need to. If you stick all the images in a centered div, you might find it works out fine?
> 
> * though that div would need to have its width set at a number of px
> * * and it would need it's position set (relative) so that its absolute children position themselves (top, right,etc) relative to the it, rather than to Background or body

Nicely!

<img width="1436" alt="Screenshot 2020-07-07 at 12 16 49" src="https://user-images.githubusercontent.com/53219789/86770371-e596bf80-c050-11ea-8d93-d89397a96b60.png">

Reply:
> styled-components definition for `Icon` above? That keeps it DRY (you heard that acronym?)

Nope but I have now :) 

Question, I've checked the documentation for styled-components and applying media queries as part of their definition but I think my implementation is a bit off here as it's sent all the icons back to their original size (i.e massive)

<img width="1098" alt="Screenshot 2020-07-07 at 14 51 19" src="https://user-images.githubusercontent.com/53219789/86784375-b89ed880-c061-11ea-8791-881b3e70dd9c.png">

<img width="1049" alt="Screenshot 2020-07-07 at 14 54 28" src="https://user-images.githubusercontent.com/53219789/86784429-c9e7e500-c061-11ea-9e2e-bd4bba853d80.png">

Reply:
Not sure if this is possible since we're using a styled-system responsive array, instead I added a variable before the `return` statement just to make it a bit more DRY. if you're free to point me in the right direction, then lmk!